library/src/com/astuetz/PagerSlidingTabStrip.java

```diff
@@ -182,6 +180,7 @@ public PagerSlidingTabStrip(Context context, AttributeSet attrs, int defStyle) {
         indicatorColor = a.getColor(R.styleable.PagerSlidingTabStrip_pstsIndicatorColor, indicatorColor);
         underlineColor = a.getColor(R.styleable.PagerSlidingTabStrip_pstsUnderlineColor, underlineColor);
         dividerColor = a.getColor(R.styleable.PagerSlidingTabStrip_pstsDividerColor, dividerColor);
+        tabTextColorInactive = a.getColor(R.styleable.PagerSlidingTabStrip_pstsTextColorInactive, tabTextColorInactive);
```

Comment:
I'd move this below (replacing the removed lines), so that it doesn't split up the attributes for the divider (color/width)